### ISSUE TITLE
Download of giggle index from Cistrome website: 404 Not Found

### ISSUE THREAD

**Issue by Telogen** [liulab-dfci/MAESTRO#151]
Hi team! I'm following your tutorial and trying to run `ATACAnnotateChromatinAccessibility()`, before that I have to download giggle index from Cistrome website, but the link `http://cistrome.org/~galib/giggle.all.tar.gz` does not exist anymore, could you please offer the new link of giggle index? Many thanks!
